test/optimization/common.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Shared functionality and helpers for the unit tests."""
+
+from enum import Enum
+import inspect
+import logging
+import os
+import unittest
+import time
+
+from qiskit.optimization import __path__ as qiskit_optimization_path
+
+
+class Path(Enum):
+    """Helper with paths commonly used during the tests."""
+    # Main SDK path:    qiskit/optimization/
+    SDK = qiskit_optimization_path[0]
+    # test.python path: test/optimization
+    TEST = os.path.dirname(__file__)
+
+
+class QiskitOptimizationTestCase(unittest.TestCase):
```

Comment:
The common.py under optimization doesn't exist anymore. This was all refactored and what remained was renamed to optimization_test_case.py. It seems to have been lost when merging from master. It is important to reinstate optimization_test_case.py because there is a common logging framework and common methods shared by everybody from the base class.